backend/api/urls.py

```
@@ -0,0 +1,21 @@
+from django.urls import path, include
```

Ruff (0.8.2)

`django.urls.path` imported but unused

Remove unused import

(F401)